app/metrics.yaml

```diff
@@ -1355,6 +1355,36 @@ metrics:
       - perf-android-fe@mozilla.com
       - mcomella@mozilla.com
     expires: "2021-08-11"
+  android_autofill_supported:
```

Comment:
Oh, okay, that makes sense. I'll move them. I somehow thought metrics we collect once at startup need to be in that category :)

Reply:
nope. Well, it makes the code a bit harder, because right now it's all in the block for the `metrics` object there, but really it will be just 2 more lines outside of that block then.